libs/net/entities/UserNotification.cs

```diff
@@ -37,6 +37,12 @@ public class UserNotification : AuditColumns
     /// </summary>
     [Column("resend")]
     public ResendOption? Resend { get; set; }
+
+    /// <summary>
+    /// get/set - Whether the user is subscribed to this report.
+    /// </summary>
+    [Column("is_subscribed")]
+    public bool IsSubscribed { get; set; }
```

Comment:
This will eventually provide the flag for whether a user is subscribed to a report/notification.  This way we keep a record of who updated this subscription last.